docs/content/config/security/ssl.md

```diff
@@ -119,41 +118,41 @@ You don't have to do anything else. Enjoy!
 
 2. Add a volume for `docker-mailserver` that maps the _local `certbot/certs/` folder_ to the container path `/etc/letsencrypt/`.
 
-!!! example
-
-    Add these additions to the `mailserver` service in your [`docker-compose.yml`][github-file-compose]:
-
-    ```yaml
-    services:
-      mailserver:
-        # For the FQDN 'mail.example.com':
-        hostname: mail
-        domainname: example.com
-        environment:
-          - SSL_TYPE=letsencrypt
-        volumes:
-          - ./docker-data/certbot/certs/:/etc/letsencrypt
-    ```
+    !!! example
+    
+        Add these additions to the `mailserver` service in your [`docker-compose.yml`][github-file-compose]:
+    
+        ```yaml
+        services:
+          mailserver:
+            # For the FQDN 'mail.example.com':
+            hostname: mail
+            domainname: example.com
+            environment:
+              - SSL_TYPE=letsencrypt
+            volumes:
+              - ./docker-data/certbot/certs/:/etc/letsencrypt
+        ```
     
 3. The certificate setup is complete, but remember _it will expire_. Consider automating renewals.
 
-!!! tip "Renewing Certificates"
-
-    When running the above `certonly --standalone` snippet again, the existing certificate is renewed if it would expire within 30 days.
-
-    Alternatively, Certbot can look at all the certificates it manages, and only renew those nearing their expiry via the [`renew` command][certbot::renew]:
-
-    ```sh
-    # This will need access to port 443 from the internet, adjust your firewall if needed.
-    docker run --rm -it \
-      -v "${PWD}/docker-data/certbot/certs/:/etc/letsencrypt/" \
-      -v "${PWD}/docker-data/certbot/logs/:/var/log/letsencrypt/" \
-      -p 80:80 \
-      -p 443:443 \
-      certbot/certbot renew
-    ```
-
-    This process can also be [automated via _cron_ or _systemd timers_][certbot::automated-renewal].
+    !!! tip "Renewing Certificates"
+    
+        When running the above `certonly --standalone` snippet again, the existing certificate is renewed if it would expire within 30 days.
+    
+        Alternatively, Certbot can look at all the certificates it manages, and only renew those nearing their expiry via the [`renew` command][certbot::renew]:
+    
+        ```sh
+        # This will need access to port 443 from the internet, adjust your firewall if needed.
+        docker run --rm -it \
+          -v "${PWD}/docker-data/certbot/certs/:/etc/letsencrypt/" \
+          -v "${PWD}/docker-data/certbot/logs/:/var/log/letsencrypt/" \
+          -p 80:80 \
+          -p 443:443 \
+          certbot/certbot renew
+        ```
+    
+        This process can also be [automated via _cron_ or _systemd timers_][certbot::automated-renewal].
```

Comment:
While similar, this admonition is not associated to the third ordered list item prior to it.

This is discussing renewal in general with a brief mention at the end about automating the otherwise manually invoked renewal.

The ordered list item is only about advising to consider automating renewal. Although I guess you could associate the admonition to it..?

---

The admonition prior was indented as it's associated via an example, rather than a tip.

Reply:
Alright. Figured this belongs together.